packages/core/src/render3/jit/directive.ts

```diff
@@ -156,6 +179,54 @@ export function compileComponent(type: Type<any>, metadata: Component): void {
   });
 }
 
+function getStandaloneDependencies(imports: Type<any>[]): R3TemplateDependencyFacade[] {
+  const dependencies: R3TemplateDependencyFacade[] = [];
+  for (const rawDep of imports) {
+    const dep = resolveForwardRef(rawDep);
+    if (!dep) {
+      // TODO: real error
+      throw new Error(`ForwardRef issue?`);
+    }
+    const ngModuleDef: NgModuleDef<any>|null = getNgModuleDef(dep);
```

Comment:
```suggestion
    const ngModuleDef = getNgModuleDef<Type<unknown>>(dep);
```
Should this have a more restrictive type?